__tests__/bundle/info-override/.redocly.yaml

```diff
@@ -0,0 +1,8 @@
+apis:
+  main:
+    root: ./main.yaml
+
+decorators:
+  info-override:
+    title: Updated title
+    x-vendor: custom extension
```

Comment:
Should  there be a descriptions like in snapshot ?

 ```yaml
 description: |
    This is a test API description.

    # Heading test

    Body [test](#) content.
```

Reply:
You are right

Reply:
From the docs it looks like it should not need it. It overrides properties and leaves the ones that were not in the override alone, right?

Reply:
Nevermind, the original and new one have different descriptions. 🤣

Reply:
Should be resolved now.